src/affiliations/views.py

```diff
@@ -23,3 +24,90 @@ class AffiliationsDetail(generics.RetrieveUpdateDestroyAPIView):
     permission_classes = [permissions.IsAuthenticatedOrReadOnly]
     queryset = Affiliation.objects.all()
     serializer_class = AffiliationSerializer
+
+
+@api_view(["GET"])
```

Comment:
Would it be possible to add some automated tests for these API routes? Something basic along the lines of https://www.django-rest-framework.org/api-guide/testing/#checking-the-response-data?

Reply:
Just made these changes! Let me know your thoughts!